### ISSUE TITLE
Feature request: Region select

### ISSUE THREAD

**Issue by acsch** [CARTAvis/carta#127]
It would be useful to be able to select several or all regions (eg to
remove them or to change their colour etc). 

In addition, an 'inverse' function like ds9 has could be useful: 
to select a few regions, inverse them and remove the previously
unselected ones.

**Comment by kswang1029** [comment]
region multi-selection is part of the upcoming region enhancement plan (under internal discussion).

**Comment by Jordatious** [comment]
Yes, we're currently discussing this @acsch, and I agree with your suggestions. It would be helpful if you could outline your use case. How many regions are you loading and where do they come from?

**Comment by acsch** [comment]
At the moment I'm working with my own regions (set by searching the cube 
for sources). The number can be anything from a several to hundreds (when 
all region from the full frequency range are used and we have the final 
deep cube - no idea how many sources to expect). Use cases would be 
various:

(a) loading/removing sources: I may want to compare my own source list 
with one from someone else - so I load both, but then I want to go back to 
only see my onw (eg to continue searching). Currently I have to click on 
each source and then remove it. Instead, I could select all, remove all 
and then relaod my onw source list.

(b) check on cross matches: I could load a region from an optical 
catalogue, select the ones I see matching with a source in the cube, then 
remove all the umnmatched sources.

(c) compare two lists: mostly I do not bother setting the colour for a 
region. But for a comparison, it would be good to have two lists in 
different colour. If the files, as I said, are in the same colour, I could 
load one set, select it all and change the colour, then load the second 
set.

That's all I can think of at the moment, but mostly what I wanted recently 
is remove all sources (since I can't hide them).
